### ISSUE TITLE
Execution error when executing rule SENDIG-13 when Excel contains a dataset that is not in the SEND standard

### ISSUE THREAD

**Issue by JozefAerts** [cdisc-org/cdisc-rules-engine#497]
When executing rule SENDIG-13 (Expected variables must be included) in the rule editor, and the test Excel contains a dataset that is not in the (SEND) standard, e.g. "AE", the engine reports an "execution error" for that dataset. 
I think that in that case, it should skip the dataset.
I attach the export from rule editor (using the "export artifacts") which contains import and generated files.
P.S. This rule uses the CDISC library, so the execution error probably is generated when the engine asks which variables are the "expected" variables for the (AE) domain, which however is not in the library, in this case, AE is not in SEND.
[unit-test-coreid-SEND13-positive_execution_error_AE.zip](https://github.com/cdisc-org/cdisc-rules-engine/files/12248649/unit-test-coreid-SEND13-positive_execution_error_AE.zip)

**Comment by SFJohnson24** [comment]
[Rule(26).zip](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK29sQ1E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--b27ba300a4b629a780257044b865afb65e54c13d/Rule%2826%29.zip)
It looks like this rule was published last month.  I ran your dataset again against SEND13 and there is no more error for AE.  It does however list AE as 'successful' but without issues.  Is this still an issue or can we close this ticket?

**Comment by JozefAerts** [comment]
Yes, I tested it again too, and for AE (which does not exist in SEND) it just says "successful", which is fine.
One may discuss whether it then would need to say "skipped", but that would again cause problems as the rules was merged with CG0016, TIG0301, TIG0065, with CG0016 being for SDTM, where CG0016 would also be applicable to AE.
So, yes, the ticket can be closed.
